Opcodes/py/pythonopcodes.h

```diff
@@ -21,7 +21,13 @@
 #ifndef _pycsound_pythonopcodes_h_
 #define _pycsound_pythonopcodes_h_
 
-#include <Python.h>
+#ifdef _DEBUG
+# undef _DEBUG
+#  include <Python.h>
+# define _DEBUG
+#else
+# include <Python.h>
+#endif
```

Comment:
Thanks for adding this, it should address linking issues I've had for python interfaces/opcodes.